default.nix

```diff
@@ -13,6 +13,7 @@ in naersk.buildPackage {
       pkgs.pkg-config
       pkgs.perl
   ];
+gitSubmodules = true;
```

Comment:
Why?

Reply:
We have recently been switching to git dependencies, so it will be good to add support by default in case we need in future

Reply:
Ah, I see, that relates to the dependencies, agreed!